napari/_event_loop.py

```diff
@@ -8,5 +8,11 @@
     def gui_qt(**kwargs):
         raise exc
 
-    def run(**kwargs):
```

Comment:
mypy expects the signatures of overriden functions to be exactly the same, hence the change.

Reply:
I was going to ask why the same isn't needed for `gui_qt`, but it's original definition has the `@contextmanager` so I guess this isn't strictly an overload here even though the name is clobbered. Given that the `gui_qt` defined just raises, I think that's fine.